Description: sdk: Delay library name config for extapps
 When building external applications the config will only
 be pulled in from a later makefile. Move the setup of
 the common library name until after that.
Forwarded: Not yet. But should be.
Author: Stefan Bader <stefan.bader@canonical.com>
Last-Update: 2015-08-12

Index: dpdk-2.0.0/mk/rte.vars.mk
===================================================================
--- dpdk-2.0.0.orig/mk/rte.vars.mk
+++ dpdk-2.0.0/mk/rte.vars.mk
@@ -73,11 +73,6 @@ ifneq ($(BUILDING_RTE_SDK),)
   endif
 endif
 
-RTE_LIBNAME := $(CONFIG_RTE_LIBNAME:"%"=%)
-ifeq ($(RTE_LIBNAME),)
-RTE_LIBNAME := intel_dpdk
-endif
-
 # RTE_TARGET is deducted from config when we are building the SDK.
 # Else, when building an external app, RTE_TARGET must be specified
 # by the user.
@@ -91,6 +86,11 @@ ifeq ($(BUILDING_RTE_SDK),)
 include $(RTE_SDK)/mk/internal/rte.extvars.mk
 endif
 
+RTE_LIBNAME := $(CONFIG_RTE_LIBNAME:"%"=%)
+ifeq ($(RTE_LIBNAME),)
+RTE_LIBNAME := intel_dpdk
+endif
+
 CONFIG_RTE_LIBRTE_E1000_PMD = $(CONFIG_RTE_LIBRTE_IGB_PMD)
 ifneq ($(CONFIG_RTE_LIBRTE_E1000_PMD),y)
   CONFIG_RTE_LIBRTE_E1000_PMD = $(CONFIG_RTE_LIBRTE_EM_PMD)
